tests/test_apply.py

```diff
@@ -678,45 +678,45 @@ def test_sample_dimensions(self, qubit_device):
         dev = qubit_device(wires=2)
         ops = [qml.RX(1.5708, wires=[0]), qml.RX(1.5708, wires=[1])]
 
-        shots = 10
+        shots = qml.measurements.Shots(10)
```

Comment:
Could we not change all of these tests to `Shots`?

```suggestion
        shots = 10
```

Reply:
Internally, the facade automatically converts them to `Shots` objects. Is that a problem?

Reply:
we should have tests for both cases :)

Reply:
I adapted the tests as agreed on slack :)